src/screens/SettingsScreen.tsx

```diff
@@ -152,19 +191,20 @@ const styles = StyleSheet.create({
         height: '100%',
     },
     headScreen: {
-        width: '100%',
+        width: '50%',
         marginTop: 40,
         marginBottom: 44,
         flexDirection: 'row',
         alignItems: 'center',
-        columnGap: 80,
+        justifyContent: 'space-between',
+        marginHorizontal: 30,
```

Comment:
marginHorizontal should me MainOffset, remove justify content, width. Wrap your title like that
![image](https://user-images.githubusercontent.com/52620425/231440290-a4794ded-cae3-4073-9b5c-8331668c3fa2.png)
so it will be centered